client/starwhale/base/bundle_copy.py

```diff
@@ -54,6 +54,18 @@ class _UploadPhase:
 
 
 class BundleCopy(CloudRequestMixed):
+    progress: Progress = Progress(
```

Comment:
we need Progress context manager : https://rich.readthedocs.io/en/stable/progress.html#advanced-usage

```
With Progress(xxx) as p:
   ...
```

Reply:
fixed